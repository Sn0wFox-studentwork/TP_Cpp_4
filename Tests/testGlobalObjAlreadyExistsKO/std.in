S a 0 0 1 1
S a 2 5 8 6
R a 2 5 8 6
PC a 0 0 2 2 6 6
OI a a
OR a a
EXIT
