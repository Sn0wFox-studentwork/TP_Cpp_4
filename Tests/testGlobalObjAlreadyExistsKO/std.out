OK
ERR
# a already exists
ERR
# a already exists
ERR
# a already exists
ERR
# a already exists
ERR
# a already exists
